packages/fxa-content-server/app/scripts/templates/partial/account-suggestion.mustache

```diff
@@ -1,6 +1,6 @@
 {{#showAccountSuggestion}}
   <div class="info mx-auto mb-5" id="suggest-account">
       {{#unsafeTranslate}}Why do I need to create this account? <a %(escapedSuggestionAttrs)s>Find out here</a>{{/unsafeTranslate}}
-      <span class="dismiss" tabindex="-1">&#10005;</span>
+      <img src="/images/close.svg" class="dismiss" tabindex="-1" alt="&#10005;" />
```

Comment:
A11y question: how would a screen reader announce `&#10005;` (✕), or should this be something more screen reader friendly like "Close"?

Reply:
First off, I just want to note that this PR is still very much a work in progress/draft, it might still change quite a bit. :) I'm still testing out a few different approaches.

In this case, I don't think a screen reader would read the alt text at all because of the tabindex of -1 - I think the original intent here was for `&#10005;` to serve as a visual fallback?

If the banner *should* be dismissable by a screen reader user, then this approach definitely needs to be modified, and I agree that we'd want the alt text to be more descriptive of the action.

Reply:
@LZoog What are your thoughts on making these dismiss 'buttons' (that are not actually buttons...) tabbable?